src/main/java/com/truelayer/java/payments/entities/CreatePaymentRequest.java

```diff
@@ -34,4 +35,9 @@ public class CreatePaymentRequest {
      * Optional field for configuring risk assessment and the payment_creditable webhook
      */
     private RiskAssessment riskAssessment;
+
+    /**
+     * Optional field for sub-merchant details
+     */
+    private SubMerchants subMerchants;
```

Comment:
Up for discussion: Do we need to also add an acceptance test for this new field? Does it make sense?